packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -120,43 +154,178 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
 
   return (
     <Box
-      className={cx(
-        'iui-expandable-block',
-        {
-          'iui-with-caption': !!caption,
-          'iui-expanded': expanded,
-          'iui-small': size === 'small',
-          'iui-borderless': styleType === 'borderless',
-        },
-        className,
+      className={cx('iui-expandable-header', className)}
+      role='button'
+      aria-expanded={isExpanded ?? !disabled}
+      aria-disabled={disabled}
+      tabIndex={0}
+      onClick={handleToggle}
+      onKeyDown={onKeyDown}
+      ref={forwardedRef}
+      {...rest}
+    >
+      {children ?? (
+        <>
+          {expandIcon && <ExpandableBlock.ExpandIcon />}
```

Comment:
typo?

```suggestion
          {expandIcon ?? <ExpandableBlock.ExpandIcon />}
```

Reply:
`??` doesn't work I think because there might not be any value but just a prop pass in like this 
![image](https://github.com/iTwin/iTwinUI/assets/22462084/eff2f923-365d-4ab1-9769-1a4f23672246)

You can see that it doesn't show up here https://itwin.github.io/iTwinUI/1354/react/?path=/story/core-expandableblock--basic

Reply:
But why would you want to pass it without a value? it's supposed to be a reactnode, not a boolean.
![](https://github.com/iTwin/iTwinUI/assets/9084735/9437e147-6d14-4524-8f1a-932afe7dfb25)

On that note, I'm unsure if we even need this prop. Is there a case where you're trying to render the header without ExpandIcon or with a custom one?

Reply:
Yea I think we talked about this
![image](https://github.com/iTwin/iTwinUI/assets/22462084/21b92c54-4213-4b2b-871a-17c35583cf82)

So I tried to make the icon customizable, but that makes the code too extra so I tried to add a convenience option as well

Reply:
Yea I think we talked about this
![image](https://github.com/iTwin/iTwinUI/assets/22462084/21b92c54-4213-4b2b-871a-17c35583cf82)

So I tried to make the icon customizable, but that makes the code too extra so I tried to add a convenience option as well

Reply:
Oh I see. So you can keep the `expandIcon: React.ReactNode` if you want. But _imo_, it is also ok to leave it - it will still be possible to customize it using children. It is not that often (if at all) that you want to change the icon, so extra code is ok. (In my original comment, what I really meant by "customizable" is adding a custom css classname, not changing the icon itself.)

Reply:
If you're keeping `expandIcon: React.ReactNode`, then all stories and examples need to be changed to remove it, because it should not be passed without a value.

Reply:
but isn't that how it is used? to be passed in so that default icon will display? Or should we change it to be `boolean` instead?

Reply:
no, basic usage should be like this:

```jsx
<ExpandableBlock>
  <ExpandableBlock.Header label='Expandable block' />
  <ExpandableBlock.Content>Content in block!</ExpandableBlock.Content>
</ExpandableBlock>
```

Reply:
I'm still confused, then `expandIcon` is not used at all?

Reply:
It should only be used if the icon needs to be changed. That's what I've been trying to say in this whole thread. See above: https://github.com/iTwin/iTwinUI/pull/1354#discussion_r1247110364

Custom usage for a ReactNode would be like this:
```jsx
<ExpandableBlock>
  <ExpandableBlock.Header label='Expandable block' expandIcon={'➡️'} />
  <ExpandableBlock.Content>Content in block!</ExpandableBlock.Content>
</ExpandableBlock>
```

And by default, it should use the caret icon (this is achieved using `??`). It doesn't make sense to ask users to always pass `expandIcon` for the default caret icon; we should include it by default because that's what is desired in 99% of cases.

Reply:
ok yea that makes sense. I have updated all stories and website example and tested working as expected